cherab/openadas/rates/atomic.pyx

```diff
@@ -97,3 +97,44 @@ cdef class NullRecombinationRate(CoreRecombinationRate):
 
     cpdef double evaluate(self, double density, double temperature) except? -1e999:
         return 0.0
+
+
+cdef class ThermalChargeExchangeRate(CoreRecombinationRate):
```

Comment:
Maybe relabel to `ThermalCXRate`. I'll raise a ticket to create the appropriate cherab-core rate object.

Reply:
[renamed](https://github.com/Mateasek/cherab_openadas/blob/e9458e956af735110764c7f8d3c7403889e7d9de/cherab/openadas/rates/atomic.pyx#L102) to:

``` python
cdef class ThermalCXRate(CoreThermalCXRate):
```